torch/testing/_internal/distributed/rpc/rpc_test.py

```diff
@@ -1811,9 +1811,9 @@ def test_rref_context_debug_info(self):
         rpc.rpc_sync(worker_name(dst_rank), set_global_rref, args=(rref1,))
 
         # barrier before check 2
+        wait_until_pending_futures_and_users_flushed()
```

Comment:
Hmm, this can make tests pass, but what about real-world applications? Do we need to expose this API, maybe as a private one (start with `_`) and with a better name. :)

Reply:
Do we expect users to care about this? It seems to me that it is part of the design that the rref operations are async and therefore "lagging" a bit behind. Do we see reasons for the users to want to wait for these and thus force a synchronization?

Reply:
No, I am not aware of any use case as of today.